workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -20,6 +20,14 @@ public class GdkToolsConfiguration
         public string DevAuthTokenFilepath => Path.Combine(DevAuthTokenFullDir, "DevAuthToken.txt");
         public int DevAuthTokenLifetimeHours => TimeSpan.FromDays(DevAuthTokenLifetimeDays).Hours;
 
+        private const string CustomSnapshotPathPrefKey = "CustomSnapshotPath";
+
+        public string CustomSnapshotPath
+        {
+            get => PlayerPrefs.GetString(CustomSnapshotPathPrefKey, "snapshots/default.snapshot");
+            set => PlayerPrefs.SetString(CustomSnapshotPathPrefKey, value);
```

Comment:
Is it worth validating that `value` is sensible here? I.e. - non-null or empty

Reply:
The value is not filled in manually, it's picked from a OpenFilePanel Unity window https://docs.unity3d.com/ScriptReference/EditorUtility.OpenFilePanel.html and the text field displaying it is disabled.

Besides, per @zeroZshadow suggestion, I'm validating if the file exists before starting the launch

`[MenuItem("SpatialOS/Local launch %l", false, MenuPriorities.LocalLaunch)]
        private static void LaunchMenu()
        {
            toolsConfig = GdkToolsConfiguration.GetOrCreateInstance();
            if (!File.Exists(toolsConfig.CustomSnapshotPath))
            {
                Debug.LogError($"Snapsthot {toolsConfig.CustomSnapshotPath} not found. Make sure the file exists and it has not been moved or renamed");
                return;
            }

            Debug.Log($"Launching SpatialOS locally with custom snapshot {toolsConfig.CustomSnapshotPath}.");
            EditorApplication.delayCall += LaunchLocalDeployment;
        }`

Reply:
> Of course, I can still add the validation if you think it's necessary

If we don't expect this valid to be set from anywhere else in the future, its probably fine as-is.